frontend/appflowy_flutter/integration_test/appearance_settings_test.dart

```diff
@@ -41,5 +42,45 @@ void main() {
 
       expect(find.textContaining('Abel'), findsOneWidget);
     });
+
+    testWidgets('reset the font family', (tester) async {
```

Comment:
the `runner.dart` doesn't include this file.

Reply:
https://github.com/AppFlowy-IO/AppFlowy/pull/3137/commits/db4514a59dff981f1d00efa2727c1aca9b451b5e